packages/x-components/src/x-modules/tagging/store/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './actions/track-tagging.action';
```

Comment:
Create an `index.ts` file in the `/actions` folder exporting it.

```suggestion
export * from './actions';
```